common/src/main/java/com/gregtechceu/gtceu/common/data/GCyMMachines.java

```diff
@@ -0,0 +1,732 @@
+package com.gregtechceu.gtceu.common.data;
+
+import com.gregtechceu.gtceu.GTCEu;
+import com.gregtechceu.gtceu.api.data.RotationState;
+import com.gregtechceu.gtceu.api.data.chemical.ChemicalHelper;
+import com.gregtechceu.gtceu.api.data.tag.TagPrefix;
+import com.gregtechceu.gtceu.api.machine.MachineDefinition;
+import com.gregtechceu.gtceu.api.machine.MultiblockMachineDefinition;
+import com.gregtechceu.gtceu.api.machine.multiblock.CoilWorkableElectricMultiblockMachine;
+import com.gregtechceu.gtceu.api.machine.multiblock.PartAbility;
+import com.gregtechceu.gtceu.api.machine.multiblock.WorkableElectricMultiblockMachine;
+import com.gregtechceu.gtceu.api.pattern.FactoryBlockPattern;
+import com.gregtechceu.gtceu.api.pattern.Predicates;
+import com.gregtechceu.gtceu.api.recipe.OverclockingLogic;
+import com.gregtechceu.gtceu.common.machine.multiblock.part.ParallelHatchPartMachine;
+import net.minecraft.network.chat.Component;
+
+import static com.gregtechceu.gtceu.api.GTValues.*;
+import static com.gregtechceu.gtceu.api.machine.multiblock.PartAbility.*;
+import static com.gregtechceu.gtceu.api.pattern.Predicates.*;
+import static com.gregtechceu.gtceu.api.pattern.util.RelativeDirection.*;
+import static com.gregtechceu.gtceu.api.registry.GTRegistries.REGISTRATE;
+import static com.gregtechceu.gtceu.common.data.GCyMBlocks.*;
+import static com.gregtechceu.gtceu.common.data.GCyMRecipeTypes.ALLOY_BLAST_RECIPES;
+import static com.gregtechceu.gtceu.common.data.GTBlocks.*;
+import static com.gregtechceu.gtceu.common.data.GTMachines.registerTieredMachines;
+import static com.gregtechceu.gtceu.common.data.GTMaterials.NaquadahAlloy;
+import static com.gregtechceu.gtceu.common.data.GTRecipeTypes.*;
+
+/**
+ * @author Rundas
+ * @implNote Gregicality Multiblocks
+ */
+public class GCyMMachines {
+    public static void init() {}
+
+    public static final MachineDefinition[] PARALLEL_HATCH = registerTieredMachines("parallel_hatch",
+            ParallelHatchPartMachine::new,
+            (tier, builder) -> builder
+                    .langValue(switch (tier) {
+                        case 5 -> "Elite";
+                        case 6 -> "Master";
+                        case 7 -> "Ultimate";
+                        case 8 -> "Super";
+                        default -> "Simple"; // Should never be hit.
+                    } + " Parallel Control Hatch")
+                    .rotationState(RotationState.ALL)
+                    .abilities(PartAbility.PARALLEL_HATCH)
+                    .workableTieredHullRenderer(GTCEu.id("block/machines/parallel_hatch_mk" + (tier - 4)))
+                    .tooltips(Component.translatable("gtceu.parallelhatch_mk"+tier))
```

Comment:
this is a typo, no?

Reply:
also for the lang key you could do `"gtceu.tooltip.parallel_hatch_mk" + (tier - 4)`